packages/framework/src/Model/Pricing/Group/PricingGroupRepository.php

```diff
@@ -92,7 +92,7 @@ public function existsUserWithPricingGroup(PricingGroup $pricingGroup)
     {
```

Comment:
I would rename function `existsUserWithPricingGroup` to `existsCustomerUserWithPricingGroup`